### ISSUE TITLE
Segmentation fault stringtie v2.0.1

### ISSUE THREAD

**Issue by fgypas** [gpertea/stringtie#236]
Hi @mpertea 

I have another issue with stringtie (v 2.0.1) with PACbio reads.

I run the following command:

```
stringtie \
debug_segmentation_stringtie_v2.0.1.bam \
-G Homo_sapiens.GRCh38.90.chr.gtf \
-o debug_segmentation_stringtie_v2.0.1.gtf \
-A debug_segmentation_estimates_v2.0.1.tsv \
-L \
-p 1 \
-v
```

and I get a segmentation fault.

> Default stack size for threads: 8388608
> [09/02 14:55:22]>bundle 1:11869-2036940 [28289 alignments (15281 distinct), 2990 junctions, 640 guides] begins processing...
> [09/02 14:55:23]^bundle 1:11869-2036940 done (324 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:2050470-2220738 [75 alignments (26 distinct), 1 junctions, 45 guides] begins processing...
> [09/02 14:55:23]^bundle 1:2050470-2220738 done (2 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:2228695-2391707 [528 alignments (231 distinct), 21 junctions, 19 guides] begins processing...
> [09/02 14:55:23]^bundle 1:2228695-2391707 done (11 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:2391775-2801717 [1285 alignments (302 distinct), 62 junctions, 93 guides] begins processing...
> [09/02 14:55:23]^bundle 1:2391775-2801717 done (9 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:2859201-3016622 [8 alignments (8 distinct), 1 junctions, 2 guides] begins processing...
> [09/02 14:55:23]^bundle 1:2859201-3016622 done (1 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:3025819-3069001 [1 alignments (1 distinct), 1 junctions, 8 guides] begins processing...
> [09/02 14:55:23]^bundle 1:3025819-3069001 done (0 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:3069168-3438621 [327 alignments (150 distinct), 33 junctions, 14 guides] begins processing...
> [09/02 14:55:23]^bundle 1:3069168-3438621 done (12 processed potential transcripts).
> [09/02 14:55:23]>bundle 1:3489920-3930790 [2398 alignments (1036 distinct), 209 junctions, 88 guides] begins processing...
> [09/02 14:55:23] 30513 aligned fragments found.
> Segmentation fault

I extracted the coordinates that cause the issue. You can find a debug test file here:
https://drive.google.com/open?id=1AzHLZ1oBiB_xYjYx_cneZaX1T_nKGsbf

The annotation I use is the same as in #235 (ENSEMBL v90, no preprocessing)

I use the pre-combiled version of stringtie (https://github.com/gpertea/stringtie/releases/download/v2.0.1/stringtie-2.0.1.Linux_x86_64.tar.gz)

Thank you in advance for your help and let me know if you need additional information. 

Best
Foivos

**Comment by fgypas** [comment]
Please note that in v 2.0 I do not get this error.

**Comment by gpertea** [comment]
Thank you so much for the swift report and providing the data - indeed we found the problem introduced by the "fix" implemented in v2.0.1 (sorry about that), so now hopefully v2.0.2 should address that (and earlier) problems.
This latest v2.0.2 version can be now downloaded from https://github.com/gpertea/stringtie/releases or http://ccb.jhu.edu/software/stringtie/#install

**Comment by fgypas** [comment]
Thank you for the quick fix. I am closing the issue.
